lacci/lib/shoes/drawables/para.rb

```diff
@@ -25,7 +25,11 @@ def initialize(*args, stroke: nil, size: :para, font: nil, hidden: false, **html
     end
 
     def text_children_to_items(text_children)
-      text_children.map { |arg| arg.is_a?(String) ? arg : arg.linkable_id }
+      if text_children.is_a?(Array)
```

Comment:
Can you say more about the use case here?

Reply:
in the example i rfered in previous comment we are passing [shape.text] to para..?

Reply:
So you mean list_box-shape-report uses it as a string? That's interesting. If so we should support that case, but then we should probably also support the case where it uses a list of TextDrawables.

Reply:
> So you mean list_box-shape-report uses it as a string? 
i think so, i am not totally sure but i cant really imagine any other way for it to work.

>That's interesting. If so we should support that case, but then we should probably also support the case where it uses a list of TextDrawables.
hm didnt got this -- something like? ["a","b"]?

Reply:
So, I don't *know* of anything using this. I was asking if the list_box-shape-report thing was what you meant, because you move that example into working in this same PR (though it should probably go in the corresponding place in the "legacy" directory, not into top-level examples.)

But basically, I'm trying to figure out what you're doing. I don't think old Shoes has this API in this form. If the example also doesn't have it, and you didn't add an example for it, I'm confused about the intent.